gef-ui/server/index.js

```diff
@@ -92,15 +89,13 @@ app.use(healthcheck);
 app.use('/', routes);
 
 app.use('/assets', express.static(path.join(__dirname, '..', 'public')));
-// eslint-disable-next-line
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Could we just remove the unused var instead?

Reply:
sorted